src/lib/pickles/wrap_verifier.ml

```diff
@@ -1,67 +1,60 @@
-module type Inputs = Intf.Wrap_main_inputs.S
-
 module S = Sponge
-open Backend
-open Core_kernel
-open Util
 module SC = Scalar_challenge
-open Pickles_types
-open Plonk_types
-open Tuple_lib
-open Import
```

Comment:
I think `Import` was intended to be opened, why not open it here?

Reply:
I find the global `open Import` makes it confusing to know syntactically where the values come from. In particular, there's a local Scalar_challenge and one through `Import` so that when you're on line 152 or 805 and you have forgotten the 10 open, there's no way you can tell by *only* reading which `Scalar_challenge` the code actually talks about.

There are around 30 uses of module Import, 10% of it is from function type signatures which I think should be in the `mli` only if they are exported, and 10% just to use the debug flag ... so that I felt 20 uses for 1000 lines of code was okay to remove globally open scopes.

 Since `Import` clearly does not seal the interaction between Pickles and the outside world, I'm not even sure it should actually exist  in its current incarnation.

As for Core_kernel, if it's meant to *always* be open, we should make it so through `dune` but I'm not convinced there's value in the current `Import` module.

Reply:
These kinds of changes have been removed from the current version of the PR. Hence I'm resolving this discussion.